feat(*): add ERC20 standard

DZariusz opened the pull request:
Add `Salable` contract that uses ERC20 token standard.

`Salable.sol` contract has same top interface as `Sales.sol` but instead `HumanStandardToken` it uses open source `ERC20` standard. This contract can be used as standalone (same as `Sales`) and it can be used by `StakingBank` to manage staking tokens, which is basically a reason why we need `Salable` in in a first place.

LIT-195

avoloshko commented on contracts/Salable.sol:
```diff
@@ -0,0 +1,144 @@
+    frozen = !frozen;
+  }
+
+  function approveAndCall(address _stakingBank, uint256 _value) public returns (bool) {
```
can be any contract who wan't to integrate our token. not necessarily should be the Staking Bank.

avoloshko commented on migrations/deployers/Salable.js:
```diff
@@ -0,0 +1,58 @@
+        'address',
+        'uint256',
+        'string',
+        'uint8',
```
why do we need that?

DZariusz replied:
this is already present in `Sales` contract, this is nice feature that might help us to automatically deploy contract... that's why I kept it.

avoloshko commented on migrations/deployers/Salable.js:
```diff
@@ -0,0 +1,58 @@
+      wallet,
+      config['total'],
+      config['name'],
+      config['decimals'],
```
why do we need that?

DZariusz replied:
same as https://github.com/luciditytech/token-sale-contracts/pull/5#discussion_r263503280

DZariusz committed:
feat(*): `Salable` ERC20 based contract

Add `Salable` contract that uses ERC20 token standard.

LIT-195

DZariusz committed:
test(*): tests for `Salable.sol`

- unit tests for `Salable` contract
- change a way how configuration is stored
- deployer script that can be used outside repository
- add changelog

LIT-195

DZariusz committed:
update HST with Solidity v5 call()
